src/main/java/uk/gov/register/core/Entry.java

```diff
@@ -16,20 +16,28 @@
 import uk.gov.register.views.representations.CsvRepresentation;
 
 import java.time.Instant;
+import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
 
 @JsonIgnoreProperties(ignoreUnknown=true)
 @JsonPropertyOrder({"entry-number", "entry-timestamp", "item-hash", "key"})
 public class Entry implements CsvRepresentationView<Entry> {
     private final int entryNumber;
-    private final HashValue hashValue;
+    private final List<HashValue> hashValues;
     private final Instant timestamp;
     private String key;
 
     public Entry(int entryNumber, HashValue hashValue, Instant timestamp, String key) {
         this.entryNumber = entryNumber;
-        this.hashValue = hashValue;
+        this.hashValues = Arrays.asList(hashValue);
```

Comment:
You won't be able to add to the List returned by Arrays.asList(). We should create a new instance of List here.